.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+### Link to JIRA
+
+[PLAT-XYZ](https://dominodatalab.atlassian.net/browse)
+
+### What issue does this pull request solve?
+
+_placeholder_
+
+### What is the solution?
+
+_placeholder_
+
+### Testing
+Briefly describe how the change was tested. The purpose of this section is that a reviewer can identify a test gap, if any.
+
+_e.g. "I ran an upgrade from 4.2 to 4.6"._
+
+- [ ] Unit test(s)
+
+### Pull Request Reminders
+- [ ] Has relevant documentation been updated?
+- [ ] Does the code follow [Domino Python Style Guide] (https://dominodatalab.atlassian.net/wiki/spaces/ENG/pages/227180579/Domino+Python+Style+Guide)
```

Comment:
Yeah, same concern - external collaborators won't know anything about Domino Python Style Guide.

Reply:
that's true. We can default to Flake8 and Black formatting.